content/components/page-header/react/draft.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: PageHeader
+reactId: page_header
+reactStatus: draft
+---
+
+import ReactComponentPage from '~/src/components/react-component-page'
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  query PageHeaderDraftQuery($componentId: String! = "page_header", $parentPath: String! = "/components/page-header", $status: String! = "draft") {
+    ...ReactComponentInfo
+  }
+`
+
+export default function Layout({data, children}) {
+  return <ReactComponentPage data={data} showImport={false} tocItems={[{url: "#import", title: "Import"}, {url: "#examples", title: "Examples"}]}>
+    {children}
+  </ReactComponentPage>
+}
+
+```js
+import {PageHeader} from '@primer/react/drafts'
+```
+
+## Examples
```

Comment:
Same feedback for this and all other "Examples" sections: we want to eliminate these inline examples to lean more on Storybook.

Is there something blocking us from doing that? If so, what would it take to get unblocked?

Reply:
No, that's definitely possible. I just copy/pasted these from primer/react, which included hand-written examples. I'm happy to replace these with storybook embeds 👍